articles/challenge_participation.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Participating in a Challenge
+layout: article
+excerpt: Learn how to participate in challenges and submit to evaluation queues.
```

Comment:
why specifics on submitting to eval queues in the description? doesn't seem to be required terminology to challenge participation.